UnitTest1.cs

```diff
@@ -0,0 +1,157 @@
+namespace ApiTests;
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using Xunit;
+
+public class Asset
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public int FloorMapId { get; set; }
+    public double X { get; set; }
+    public double Y { get; set; }
+    public bool Active { get; set; }
+}
+
+public class AssetPositionHistory
+{
+    public int Id { get; set; }
+    public int AssetId { get; set; }
+    public int FloorMapId { get; set; }
+    public DateTime Timestamp { get; set; }
+    public double X { get; set; }
+    public double Y { get; set; }
+}
+
+
+public class ApiCrudTests : IAsyncLifetime
+{
+    private readonly HttpClient _client;
+
+    public ApiCrudTests()
+    {
+        _client = new HttpClient(new HttpClientHandler
+        {
+            ServerCertificateCustomValidationCallback = (message, cert, chain, errors) => true
+        })
+        {
+            BaseAddress = new Uri("https://localhost:7018")
+        };
+    }
+
+    public Task InitializeAsync() => Task.CompletedTask;
+
+    public Task DisposeAsync()
+    {
+        _client.Dispose();
+        return Task.CompletedTask;
+    }
+
+    [Fact]
+    public async Task AssetsCrudOperations_ShouldWorkProperly()
```

Comment:
The scope of this test is way too big. A test should only test one functionality and one use case of that functionality.
Instead of having one test for all CRUD operations, separate them into separate tests.
Add a function that resets the backend state before every test. Every test should be independent.
Look up the Arrange, Act, Assert testing pattern.
Remove comments.